fe/fe-core/src/main/antlr4/org/apache/doris/nereids/DorisParser.g4

```diff
@@ -1168,7 +1168,7 @@ nonReserved
     | MAP
     | MATCH_ALL
     | MATCH_ANY
-    | MATCH_ELEMENT_EQ
+    | MATCH_columnName
```

Comment:
what's this?

Reply:
something wrong in last commit for my git action, I will delete